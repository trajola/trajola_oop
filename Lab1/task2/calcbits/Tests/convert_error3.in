214748364899999999999