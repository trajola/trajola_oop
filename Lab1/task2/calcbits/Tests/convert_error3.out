Failed to convert '214748364899999999999' to number.
